package.json

```diff
@@ -62,6 +62,7 @@
     "rimraf": "^2.5.0",
     "run-waterfall": "^1.1.3",
     "solid-namespace": "^0.1.0",
+    "solid-permissions": "git://github.com/solid/solid-permissions.git#dz_allows_helpers",
```

Comment:
Temporary (for this pr review only), until PR https://github.com/solid/solid-permissions/pull/3 gets merged.